library/Rules/Email.php

```diff
@@ -1,10 +1,32 @@
 <?php
 namespace Respect\Validation\Rules;
 
+use Egulias\EmailValidator\EmailValidator;
```

Comment:
If you don't run `--no-dev` you cannot run PHPUnit either, I mean, you can run it, but there is no guarantee you're using the same PHPUnit we have.

There are some other dependencies yet, but, as I said before, doesn't matter the type hinting (or the use statement), the only problem you will get is if you create an instance of the object, until them PHP will not care about declarations.

You can have many `use` statement, PHP will only complain about them when you create an instance (when the autoloader is called).